vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -232,6 +232,26 @@ function Get-PrivatePackagesToCacheOnVHD {
     }
 }
 
+function Get-ToolsToVHD {
+    $cacheDir = "c:\toolscache"
+    if (!(Test-Path -Path $cacheDir)) {
+        New-Item -ItemType Directory -Path $cacheDir | Out-Null
+    }
+
+    $toolsDir = "c:\tools" # discuss
+    if (!(Test-Path -Path $toolsDir)) {
+        New-Item -ItemType Directory -Path $toolsDir | Out-Null
+    }
+
+    Write-Log "Caching du.exe"
+    $URL = "https://download.sysinternals.com/files/DU.zip"
```

Comment:
It should be better to put the URL and file list in vhdbuilder/packer/generate-windows-vhd-configuration.ps1

Reply:
Updated